engine/src/test/java/io/camunda/zeebe/engine/processing/timer/TimerStartEventTest.java

```diff
@@ -877,4 +877,68 @@ public void shouldTriggerOnlyTimerStartEvent() {
         .extracting(r -> r.getValue().getElementId())
         .containsOnly("timer_start");
   }
+
+  @Test
+  public void shouldTriggerAtSpecifiedStartTimeForTimeCycle() {
+    // given
+    // Set the start time to 10 seconds later
+    final ZonedDateTime start =
+        ZonedDateTime.ofInstant(
+                Instant.ofEpochMilli(engine.getClock().getCurrentTimeInMillis()),
+                ZoneId.systemDefault())
+            .plusSeconds(10);
+
+    final long dueDate = start.plusSeconds(10).toInstant().toEpochMilli();
+    final BpmnModelInstance model =
+        Bpmn.createExecutableProcess("process")
+            .startEvent("start")
+            .timerWithCycle(String.format("R1/%s/PT10S", start.toInstant().toString()))
+            .endEvent("end")
+            .done();
```

Comment:
:wrench: We could use a different duration for the start and the interval. Currently, the start is 10 seconds in the future and the interval is also 10 seconds. 

The test may pass if there is an issue in the implementation, for example, it uses only the interval but ignores the start. 

We could use a higher duration for the start to cover more bugs.